generatedocs/data/params.json

```diff
@@ -447,6 +447,18 @@
     "setscript.script": [
         "Load script for the app (written as a string)."
     ],
+    "setscriptvar.name" : [
+        "Name of variable to set. Will overwrite any existing variable with same name."
+    ],
+    "setscriptvar.type" : [
+        "Type of the variable.",
+        "`string`: Variable of type string e.g. `my var value`.",
+        "`int`: Variable of type integer e.g. `6`.",
```

Comment:
No need for floats?

Reply:
was thinking of it, do we? as a specific type that is.

Reply:
Don't know tbh. Just seems like it's missing. Probably someone will request it at some point, along with a multiplication/division operator..

Reply:
mm and will get messier with those kind of functions, so thought I'd keep it simple for now until someone request it with an actual case. You think we should add it already now?

Reply:
Meh let's keep it this way for now then

Reply:
also first felt it was missing, but change my mind to start simpler. Have no problem of adding it later if there's a request for it.